3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -29,3 +30,5 @@ rollTheDices()
 
 // ! Do not change or remove the code below
 module.export = rollTheDices;
+
+/* Because we do not indicate for the promise to stop at the specific event of rolling off the table, so it just keeps executing */
```

Comment:
Correct. I will also add that the promises don't talk to each other and are not aware what other dice are doing